packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -217,6 +218,11 @@ export class FluidCache implements IPersistedCache {
 
 			const currentTime = new Date().getTime();
 
+			let size: number | undefined;
+			try {
+				size = JSON.stringify(value).length;
```

Comment:
Can you profile this on larger data to see if this computation is CPU intensive?

Reply:
I also am not sure if value is serializable if I remember correctly, so the error + catch case may fire a lot which comes with overhead on some browsers

Reply:
@vladsud Don't we always add json serializable value to the persisted cache? @christiango on your side, do you also put values in the cache which are not json serializable? I added this in put instead of get because on FF side, we don't execute put on critical paths or wait for it.

Reply:
That's a very good question. MFS actually complained RE our behavior - they say we have non-serializable content.
I believe it's array buffers (blobs) - see ISnapshotContents.
MFS asked us to build an adapter that they can use to serialize this data back and forth, to be used on mobile devices where they use something else for storage (not indexDB).

Given that we want to be as efficient as possible here, and assuming that indexDB does the best job directly, I feel like we need to continue to support existing flow as is. 

That will definitely add cost and not work for snapshots, so value of this change is low.

Sorry I did not think about it, but probably we should punt on this task, unless we can see some better flow?

We could correlate these events with other events in the driver that tell the size of payloads retrieved here.
So maybe instead of doing this work, we need to ensure that events like "ObtainSnapshot_end" produce more verbose data (similar to TreesLatest_end) that tells us more about content size (or measures that correlate with it, like encodedBlobSize, blobs, trees).